sysutils/z/Portfile

```diff
@@ -13,8 +14,27 @@ platforms           darwin
 license             WTFPL-2
 supported_archs     noarch
 
-checksums           rmd160  64173ef1b8e77f1db4ee53486f55159ed8b43d83 \
-                    sha256  a06d8d56bd8373229b1f11690ac5d9f13753215ce57488efa1c5a6efc6f735bb
+if {${name} eq ${subport}} {
+   github.setup       rupa z 1.11 v
+   revision           0
+
+   checksums          rmd160  64173ef1b8e77f1db4ee53486f55159ed8b43d83 \
+                      sha256  a06d8d56bd8373229b1f11690ac5d9f13753215ce57488efa1c5a6efc6f735bb
```

Comment:
The `size` should be added here too.

Reply:
the size was missing from the original port, i didn't think i was supposed to add it. ill do it rigth away.

Reply:
Thank you. Sometimes we have to make changes to get up to our current standards for Portfiles.